### DISCUSSION TITLE
Do the problem matrix size or coefficient ranges provide an indication of number of iterations required to solve the model?

### DISCUSSION THREAD

**Discussion by hbshrestha** [ERGO-Code/HiGHS#2052] [Q&A] [answered]
I am currently solving a mps file using HIGHS solver with the following script in Python. I have a few intuitive questions in this regard. 

```
import highspy
import time

#read model
filename = 'MAINa_tes.mps'
h.readModel(filename)

#run the model
h.run()
```

1. Since I didn't assign any solver settings, does HIGHS use simplex method/algorithm as default? 

2. The coefficient ranges of the matrix is given in the image below. There are 2.25 million rows, 2.12 million columns, and 33 million elements after presolve. With this large problem size, will the point interior method solve the problem faster? (I am planning to set run_crossover to off, which I believe would require fewer iterations.)
![image](https://github.com/user-attachments/assets/d761d660-f1b9-45c7-8f53-69a6aaef364f)

3. Currently, the model has been running with over 400,000 iterations for about 8 hours. From the problem matrix or size, is it possible to get an indication of how many iterations are required or how much time is required for the model run to complete?
![image](https://github.com/user-attachments/assets/d7df4c41-d5a4-4fe5-9a7f-108bdeadd6af)

**Comment by jajhall** [comment]
For a problem this size, with no information about an optimal solution, interior point is likely to be much faster then simplex.

**Comment by jajhall** [comment]
Setting crossover to off avoids simplex-like crossover iterations.

**Comment by jajhall** [comment, marked as answer]
It's not possible to assess the runtime from the model data

**Reply by hbshrestha** [reply]
Thank you for the response!
